server/src/main/java/com/ibm/ws/lars/rest/Configuration.java

```diff
@@ -81,4 +86,25 @@ private static String computeRestBaseUri(String configString) {
         b.append("ma/v1/");
         return b.toString();
     }
+
+    /**
+     * Removes the port from a URI, if it is an HTTP or HTTPS URI and explicitly specifies the
+     * default port for that protocol.
+     *
+     * @param uri the URI
+     * @return either {@code uri} with the port removed, or {@code uri} unchanged.
+     */
+    private static URI stripDefaultPort(URI uri) {
+        if ((uri.getPort() == 80 && "http".equalsIgnoreCase(uri.getScheme()))
+            || (uri.getPort() == 443 && "https".equalsIgnoreCase(uri.getScheme()))) {
+            try {
+                // Copy uri but strip port
+                uri = new URI(uri.getScheme(), uri.getUserInfo(), uri.getHost(), -1, uri.getPath(), uri.getQuery(), uri.getFragment());
```

Comment:
If you passed in the UriInfo object here, you could create the new URI as
uriInfo.getBaseUriBuilder().port(-1).build()
Which would save copying everything around.